src/App.css

```diff
@@ -111,10 +112,9 @@ a:hover {
 }
 
 .Home-section {
-  text-align: center;
   padding-top: 1rem;
   background-color: #e0e0e0;
-  height: 80%;
+  height: 100vh;
```

Comment:
Kind remainder 😇

Reply:
Thank you very much for your suggestions